File: FinalHW-OOP/service/MiddleService.cs
namespace FinalHWOOP {
    public class MiddleService : ISmoke, IStudy {
        public void smoke() {
            Console.WriteLine("Я начинаю курить");
        }

        public void study() {
            Console.WriteLine("Ненавижу учиться");
        }

        public void leaveSchool() {
            Console.WriteLine("Я прогуливаю уроки");
        }
    }
}

File: sem7/src/main/java/ru/gb/oseminar/service/MiddleService.java
package ru.gb.oseminar.service;

import ru.gb.oseminar.interfaces.ISmoke;
import ru.gb.oseminar.interfaces.IStudy;

public class MiddleService implements ISmoke, IStudy{

    @Override
    public void study() {
        System.out.println("Я ненавижу учиться");
    }

    @Override
    public void smoke() {
        System.out.println("Я начинаю курить");
    }
    
    public void leaveSchool() {
        System.out.println("Я прогулял урок");
    }
}
